### PR TITLE
Add thin arbiter  support

### LINKED ISSUE #151: thin-arbiter volume type support
**Describe the feature you'd like to have.**
thin-arbiter support in the storage class to ensure with GCS deployment one can create a thin arbiter volume.

**What is the value to the end user? (why is it a priority?)**
https://docs.gluster.org/en/latest/Administrator%20Guide/Thin-Arbiter-Volumes/

How would the end user gain value from having this feature?
https://docs.gluster.org/en/latest/Administrator%20Guide/Thin-Arbiter-Volumes/

-----

Child of #46

### PR DESCRIPTION
**Pull request by Madhu-1**
**Describe what this PR does**
This PR adds the thin arbiter support in gluster-csi-driver

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

tested and works fine as expected

------

Fixes #151

### CODE REVIEW

**Comment by humblec** [comment]
@JohnStrunk please have a look at the SC param list and share your thought

File: pkg/glusterfs/controllerserver.go [hunk 1]
```diff
@@ -67,37 +67,77 @@ type ProvisionerConfig struct {
...
+					return nil, err
+				}
+			}
+		}
+	}
+	reqConf.gdVolReq.ReplicaCount = replicaCount
+	return &reqConf, nil
+}
 
-		default:
```
[the remark this answers is not in the document]
Comment (Madhu-1) [reply pkg/glusterfs/controllerserver.go]: done

**Comment by Madhu-1** [comment]
@JohnStrunk PTAL

**Comment by Madhu-1** [comment]
@JohnStrunk CI will fail for the static check, I will update it

**Comment by Madhu-1** [comment]
@JohnStrunk CI passed, PTAL

**Pull request merged**